### ISSUE TITLE
Configurer l'intégration Github sur Slack

### ISSUE THREAD

**Issue by Uinelj** [bdeecho/Meta#1]
On peut pas directement dire à l'intégration de poster dans un channel, il faut d'abord s'occuper de lier l'organisation et Slack.

Il serait intéressant de faire ce truc là une fois qu'on aura les différents repositories Git initialisés, histoire de pouvoir dire au bot de poster chaque information dans le bon channel, ie : 

**bdeecho/Meta** -> **#general**

**Comment by AlexFourgs** [comment]
Ok, on se penchera sur ce point une fois les différents dossiers rendues ;)

**Comment by AlexFourgs** [comment]
Je reviens sur ce point, il serait intéressant de faire comme pour Logirail, mettre un ou plusieurs bots qui indiquent les nouveautés du git (Issues & inc) automatiquement.

**Comment by Uinelj** [comment]
Yep, je m'en occupe quand je trouve du temps, mais c'était l'idée. Je ne sais pas si les bots référencent les issues aussi, par contre.

**Comment by AlexFourgs** [comment]
On en est ou ?

**Comment by Uinelj** [comment]
J'ai pas trop avancé dessus. C'est une histoire de 10 minutes, mais on a d'autres priorités.

**Comment by AlexFourgs** [comment]
Ok cool ! :)
